### PR TITLE
fix: convert http links to https in docs

### PR DESCRIPTION
**Pull request by koshachy**
docs + docs config

### CODE REVIEW

File: docs/topics/js/js-modules.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ following configurations for JavaScript modules:
     UMD modules are also able to be executed without being imported or when no module system is present. This is the default option for the `browser` and `nodejs` targets.
 - [Asynchronous Module Definitions (AMD)](https://github.com/amdjs/amdjs-api/wiki/AMD), which is in particular
    used by the [RequireJS](https://requirejs.org/) library.
-- [CommonJS](http://wiki.commonjs.org/wiki/Modules/1.1), widely used by Node.js/npm
+- [CommonJS](https://wiki.commonjs.org/wiki/Modules/1.1), widely used by Node.js/npm
```
Comment (p7nov) [inline docs/topics/js/js-modules.md:10]: This site doesn't provide https

**Review by p7nov — COMMENTED** [review]
Checked the links, one of them doesn't work with https. Although it warns that the connection is not secure, maybe it's better than nothing :)

File: docs/topics/faq.md [hunk 2]
```diff
@@ -159,7 +159,7 @@ Yes. The most active Kotlin account is [on Twitter](https://twitter.com/kotlin).
 ### Any other online Kotlin resources?
 
 The web site has a bunch of [online resources](https://kotlinlang.org/community/), including [Kotlin Digests](https://kotlin.link) by community members, 
-a [newsletter](http://www.kotlinweekly.net), a [podcast](https://talkingkotlin.com) and more.   
+a [newsletter](https://kotlinweekly.net), a [podcast](https://talkingkotlin.com) and more.
```
Comment (koshachy) [inline docs/topics/faq.md:162]:
```suggestion
a [newsletter](http://kotlinweekly.net), a [podcast](https://talkingkotlin.com) and more.   
```

File: docs/topics/js/js-modules.md (hunk 1 above)
Comment (koshachy) [inline docs/topics/js/js-modules.md:10]:
```suggestion
- [CommonJS](http://wiki.commonjs.org/wiki/Modules/1.1), widely used by Node.js/npm
```

Comment (koshachy) [reply docs/topics/js/js-modules.md]: Fixed

**Comment by koshachy** [comment]
> Checked the links, one of them doesn't work with https. Although it warns that the connection is not secure, maybe it's better than nothing :)

Yeah, there are four-five such links and I tried to omit them.
It seems that I missed this one.
I rollbacked and re-checked the links %)

**Review by p7nov — APPROVED** [review]

**Pull request merged**